src/themes/custom/app/request-copy/email-request-copy/email-request-copy.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import {
+    EmailRequestCopyComponent as BaseComponent
+} from 'src/app/request-copy/email-request-copy/email-request-copy.component';
+
+@Component({
+    selector: 'ds-email-request-copy',
+    // styleUrls: ['./email-request-copy.component.scss'],
+    styleUrls: [],
+    // templateUrl: './email-request-copy.component.html',
+    templateUrl: './../../request-copy/email-request-copy/email-request-copy.component.html',
```

Comment:
Same here. This path also looks wrong...it should point at the HTML file under /src/app/, but it never even makes it up to the /src/app folder.